mvpa2/algorithms/searchlight_hyperalignment.py

```diff
@@ -303,12 +304,13 @@ def _proc_block(self, block, datasets, measure, qe, seed=None, iblock='main'):
                         I += roi_feature_ids
                         J += [roi_feature_ids[f2]] * len(roi_feature_ids)
                         V += hmappers[isub][0]['proj'][:, f2].tolist()
-                proj = coo_matrix((V, (I, J)),
-                                  shape=(max(self.nfeatures, max(I)+1), max(self.nfeatures, max(J)+1)),
-                                  dtype=self.params.dtype)
+                proj = coo_matrix(
+                    (V, (I, J)),
+                    shape=(max(self.nfeatures, max(I) + 1), max(self.nfeatures, max(J) + 1)),
+                    dtype=self.params.dtype)
                 proj = proj.tocsc()
                 # Cleaning up the current subject's projections to free up memory
-                hmappers[isub, ] = [[]]*hmappers.shape[1]
+                hmappers[isub, ] = [[] for _ in xrange(hmappers.shape[1])]
```

Comment:
Better be safe than sorry ;-) more explicit could be (if it didn't matter) `[None]*...` or smth like that which would have forbidden inplace changes then

On March 28, 2016 1:48:21 PM EDT, Swaroop Guntupalli <notifications@github.com> wrote:
>>                  proj = proj.tocsc()
>>                  # Cleaning up the current subject's projections to
>free up memory
>> -                hmappers[isub, ] = [[]]*hmappers.shape[1]
>> +                hmappers[isub, ] = [[] for _ in
>xrange(hmappers.shape[1])]
>
>I think it is irrelevant. I am making them empty lists to remove data.
>
>---
>You are receiving this because you authored the thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/swaroopgj/PyMVPA/pull/3/files/e2f1ebea798f5d5a8a09625f4b1bd79b5f92dda1#r57603849